src/Button.re

```diff
@@ -12,19 +12,21 @@ let make =
       ~children,
     ) =>
   <button
-    onClick={GenericHelpers.optionFn(onClick)}
+    ?onClick
     type_
     disabled
     className={
       ButtonStyles.button(~variant, ~theme?, icon, iconPosition, ())
       ++ " "
       ++ className
     }>
-    {switch (icon) {
-     | Some(icon) =>
-       <span className={ButtonStyles.icon(iconPosition)}> icon </span>
-     | None => <Empty />
-     }}
+    {
+      switch (icon) {
```

Comment:
There are quite some whitespace changes here. It could be that our `refmt` versions are slightly different (ie. your global ReasonML version is slightly different to the one we use for the component library). Could you run the following command and push another commit?
```
bsrefmt --in-place src/**/*.re
```
Meanwhile, I'll make some changes so we're using `bsrefmt` instead of the global `refmt`.

Reply:
yup, sorry for using global refmt, done, run bsrefmt.

Reply:
Great stuff! Thanks for contributing! 🚀